content/edu/chicago.md

```diff
@@ -0,0 +1,48 @@
+---
+Title: R-Ladies Chicago Package Development Workshop
+---
+
+## Women's Package Development Workshop by [R-Ladies Chicago](https://rladieschicago.org) and [FoRwards](https://forwards.github.io)
+
+### Date: Saturday, February 23, 2018, 9:30am-4pm
+
+### Location
+
+Center for Spatial Data Science  
+5735 S Ellis Ave, 2nd Floor  
+Chicago, Illinois 60637 
+
+<iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2974.7548639089937!2d-87.60317058411151!3d41.79049637886998!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x880e293e7bd41241%3A0x5966b3036fc25171!2sSearle+Chemical+Laboratory%2C+5735+S+Ellis+Ave%2C+Chicago%2C+IL+60637!5e0!3m2!1sen!2sus!4v1544044603496" width="600" height="450" frameborder="0" style="border:0" allowfullscreen></iframe>
+ 
+### Applications
+
+Applications for this event will be opened in **late December 2018**. Please [join our Meetup](https://www.meetup.com/rladies-chicago/) for updates, and stay tuned for more!
+
+### Workshop Description
+
+Curious about package development, but not sure where to start? This workshop is for you! Run by R-Ladies Chicago, the goal of this workshop is to empower you to start contributing back to the R ecosystem with package development. We'll be using materials developed by Hadley Wickham, Jenny Bryan, and Di Cook for the [FoRwards organization](https://forwards.github.io/about/).
+
+By the end of this day-long workshop, you should know how to:
+
+- make your code into an R package, 
+- do collaborative coding with GitHub, 
+- write a vignette or an article, 
+- build a web page for your package, and
+- submit a package to CRAN. 
+
+Bring your own code that you want to make into a package, or work with our example. 
+
+### Cost and Travel Support
+Thanks to funding from the <a href="https://www.r-consortium.org/" target="_blank">R Consortium</a>, this workshop is free for participants.  Lunch will be provided.  
```

Comment:
At the moment this says nothing about travel support. Perhaps you should add a comment, e.g.
```suggestion
Thanks to funding from the <a href="https://www.r-consortium.org/" target="_blank">R Consortium</a>, this workshop is free for participants.  Lunch will be provided.  

We may be able to provided limited travel grants. Please check back here or follow Forwards or R-Ladies Chicago on Twitter for news.
```